Integrations/integration-FireEyeHX.yml

```diff
@@ -3043,7 +3134,112 @@ script:
     - contextPath: FireEyeHX.HostSets.url
       description: Host set FireEye url
       type: string
+  description: Get a list of all host sets known to your HX Series appliance
+- name: fireeye-hx-create-indicator
+  arguments:
+    - name: category
+      required: true
+      description: The indicator category
+    - name: create_text
+      description: The indicator name
+    - name: description
+      description: The indicator description
+  outputs:
```

Comment:
Please make sure the outputs match the outputs of `get-indicator` command as much as possible